src/Microsoft.AspNetCore.Http.Connections/Internal/HttpConnectionManager.Log.cs

```diff
@@ -25,15 +25,9 @@ private static class Log
             private static readonly Action<ILogger, string, Exception> _connectionTimedOut =
                 LoggerMessage.Define<string>(LogLevel.Trace, new EventId(5, "ConnectionTimedOut"), "Connection {TransportConnectionId} timed out.");
 
-            private static readonly Action<ILogger, Exception> _scanningConnections =
```

Comment:
Should we leave a comment in saying what these were?

Reply:
I like that. Can do